templates/hooks/certificate_authority/sdk_create_post_set_output.go.tpl

```diff
@@ -1,8 +1,8 @@
     if ko.Status.ACKResourceMetadata != nil && ko.Status.ACKResourceMetadata.ARN != nil {
         resourceARN := (*string)(ko.Status.ACKResourceMetadata.ARN)
-        csr, err := rm.getCertificateSigningRequest(ctx, *resourceARN)
+        csr, err := rm.getCertificateAuthorityCsr(ctx, *resourceARN)
         if err != nil {
             return nil, err
         }
-        ko.Status.CertificateSigningRequest = csr
-    }
+        ko.Status.CSR = []byte(*csr)
+    }
```

Comment:
Since we're putting this certificate in a secret, shouldn't we just replace it with an empty/nil string?

Reply:
This is the Certificate signing request (different from the certificate) and it doesn't need to be in a secret